src/test/java/com/wikia/webdriver/elements/mercury/components/Search.java

```diff
@@ -1,56 +1,42 @@
 package com.wikia.webdriver.elements.mercury.components;
 
-import com.wikia.webdriver.common.core.Assertion;
 import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.mercury.pages.SearchResultsPage;
 import com.wikia.webdriver.pageobjectsfactory.pageobject.BasePageObject;
 
 import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.NoSuchElementException;
 import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
 import org.openqa.selenium.support.FindBy;
 
 public class Search extends BasePageObject {
 
-  @FindBy(css = ".wikia-search__container input")
+  @FindBy(css = ".wikia-search__container input.side-search__input")
   private WebElement searchInput;
 
   @FindBy(css = ".wikia-search__clear")
   private WebElement clearSearchButton;
 
-  private String searchResultClass = ".wikia-search__results li.mw-content a";
-  private By loadingSearchResultsIndicator = By.cssSelector(".wikia-search__results li.loading");
+  @FindBy(css = ".wikia-search__search-icon > svg > use[*|href*='#search']")
+  private WebElement inputFieldSearchIcon;
+
+  private static final String searchSuggestionClass = ".wikia-search__suggestions li.mw-content a";
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS00115)